### PR TITLE
ms github action templating

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

**Comment by smark88** [comment]
Pending this reviewed first https://github.com/broadinstitute/datarepo-actions/pull/1 the action check outs will be changed to a new tag from ms-debian.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @ddietterich @RoriCremer @thathert @myessail @marikomedlock @fboulnois
```
Comment (thathert) [inline .github/CODEOWNERS:5]: Was this supposed to happen?

File: .github/workflows/gradle-build-pr.yml [hunk 2]
```diff
@@ -26,155 +26,72 @@ jobs:
...
+        uses: broadinstitute/datarepo-actions@0.1.0
+        with:
+          actions_subcommand: 'helmdeploy'
+          helm_secret_chart_version: '0.0.4'
+          helm_datarepo_chart_version: '0.0.8'
+      - name: "Run Integration test via Gradle"
+        uses: broadinstitute/datarepo-actions@0.1.0
+        with:
+          actions_subcommand: 'gradleinttest'
+          pgport: ${{ job.services.postgres.ports[5432] }}
```
Comment (thathert) [inline .github/workflows/gradle-build-pr.yml:86]: Why is this here? The integration tests shouldn't be touching Postgres

File: .github/workflows/gradle-build-pr.yml [hunk 3]
```diff
@@ -26,155 +26,72 @@ jobs:
...
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+      - name: "Whitelist Runner IP"
+        uses: broadinstitute/datarepo-actions@0.1.0
```
Comment (thathert) [inline .github/workflows/gradle-build-pr.yml:60]: We repeat this version string a lot of places. Is there a way we can define it once and then reuse that throughout this file?

File: .github/workflows/gradle-build-pr.yml [hunk 4]
```diff
@@ -26,155 +26,72 @@ jobs:
...
-        uses: actions/setup-java@v1
+      - name: "Run Connected test via Gradle"
+        uses: broadinstitute/datarepo-actions@0.1.0
         with:
-          java-version: 8
-      - name: "Install Gradle"
-        uses: broadinstitute/gradle-command-action@v1
-      - name: "Install Google Cloud SDK"
-        uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.1.1
+          actions_subcommand: 'gradleinttest'
```
Comment (thathert) [inline .github/workflows/gradle-build-pr.yml:32]: Maybe this should just be `gradletest` since the subcommand is used for unit, connected, and integration tests?

**Review by thathert — COMMENTED** [review]
A few questions. Where are the unit tests run?

File: .github/workflows/gradle-build-pr.yml (hunk 3 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:60]: version string ??? you mean the tag 0.1.0?

File: .github/workflows/gradle-build-pr.yml (hunk 4 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:32]: read the underlying code you pass the test to gradleinttest

File: .github/workflows/gradle-build-pr.yml (hunk 2 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:86]: Would this include ditching the postgres service for the job?

File: .github/CODEOWNERS (hunk 1 above)
Comment (smark88) [reply .github/CODEOWNERS:5]: no

File: .github/workflows/gradle-build-pr.yml (hunk 4 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:32]:
```
          actions_subcommand: 'gradleinttest'
          test_to_run: 'testIntegration'
```

File: .github/workflows/gradle-build-pr.yml (hunk 2 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:86]: fixed

File: .github/CODEOWNERS (hunk 1 above)
Comment (smark88) [reply .github/CODEOWNERS:5]: How do i go about fixing this one ?

File: .github/workflows/gradle-build-pr.yml (hunk 2 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:86]: removing that broke the test
